pkgs/development/compilers/kotlin/default.nix

```diff
@@ -1,43 +1,100 @@
 { lib, stdenv, fetchurl, makeWrapper, jre, unzip }:
 
-stdenv.mkDerivation rec {
-  pname = "kotlin";
+let
   version = "1.5.31";
+  homepage = "https://kotlinlang.org/";
+  license = lib.licenses.asl20;
+  description = "A modern programming language that makes developers happier";
+in {
+  default = stdenv.mkDerivation rec {
+    pname = "kotlin";
+    inherit version;
 
-  src = fetchurl {
-    url = "https://github.com/JetBrains/kotlin/releases/download/v${version}/kotlin-compiler-${version}.zip";
-    sha256 = "sha256-ZhERKG8+WsBqrzqUA9hp2alqF2tisUGBS+YmpHJJ/p4=";
+    src = fetchurl {
+      url = "https://github.com/JetBrains/kotlin/releases/download/v${version}/kotlin-compiler-${version}.zip";
+      sha256 = "sha256-ZhERKG8+WsBqrzqUA9hp2alqF2tisUGBS+YmpHJJ/p4=";
+    };
+
+    propagatedBuildInputs = [ jre ];
+    nativeBuildInputs = [ makeWrapper unzip ];
+
+    installPhase = ''
+      mkdir -p $out
+      rm "bin/"*.bat
```

Comment:
```suggestion
      rm bin/*.bat
```

Reply:
The suggested change of:

```
-      rm "bin/"*.bat
+      rm bin/*.bat
```

Is on unchanged code from the current HEAD of kotlin; I don't think it is appropriate to include extra unrelated to adding kotlin-native changes.